x-pack/plugins/observability_solution/investigate_app/public/pages/details/components/investigation_timeline/investigation_search_bar/investigation_search_bar.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { css } from '@emotion/react';
+import type { TimeRange } from '@kbn/es-query';
+import type { SearchBar } from '@kbn/unified-search-plugin/public';
+import React from 'react';
+import { useKibana } from '../../../../../hooks/use_kibana';
+import { InvestigationEventTypesFilter } from './investigation_event_types_filter';
+
+interface Props {
+  dateRangeFrom?: string;
+  dateRangeTo?: string;
+  onQuerySubmit: (payload: { dateRange: TimeRange }, isUpdate?: boolean) => void;
+  onRefresh?: Required<React.ComponentProps<typeof SearchBar>>['onRefresh'];
+  onEventTypesSelected: (eventTypes: string[]) => void;
+}
+
+export function InvestigationSearchBar({
+  dateRangeFrom,
+  dateRangeTo,
+  onQuerySubmit,
+  onRefresh,
+  onEventTypesSelected,
+}: Props) {
+  const {
+    dependencies: {
+      start: {
+        unifiedSearch: {
+          ui: { SearchBar },
+        },
+      },
+    },
+  } = useKibana();
+
+  return (
+    <EuiFlexGroup
+      direction="row"
+      gutterSize="s"
+      alignItems="flexStart"
+      justifyContent="spaceBetween"
+      responsive
+      css={css`
```

Comment:
Out of curiosity, why do we use the `css` emotion in this inline style instead of the built-in `style`?

Reply:
I tend to use it all the time, just in case I need to write nested selectors. But true it is not needed here, and style would work the same